src/Features/CSharp/Portable/ConvertLinq/ConvertForEachToLinqQuery/AbstractToMethodConverter.cs

```diff
@@ -189,7 +190,7 @@ protected InvocationExpressionSyntax CreateInvocationExpression(ExpressionSyntax
             => SyntaxFactory.InvocationExpression(
                     SyntaxFactory.MemberAccessExpression(
                         SyntaxKind.SimpleMemberAccessExpression,
-                        SyntaxFactory.ParenthesizedExpression(queryOrLinqInvocationExpression),
+                        SyntaxFactory.ParenthesizedExpression(queryOrLinqInvocationExpression).WithAdditionalAnnotations(Simplifier.Annotation),
```

Comment:
alternatively.  do `queryOrLinqInvocationExpression.Parenthesize()`, which should do this.